tests/benchmark/query/execution.cpp

```diff
@@ -32,30 +32,30 @@
 #include "query/interpreter.hpp"
 #include "storage/v2/storage.hpp"
 
-// The following classes are wrappers for utils::MemoryResource, so that we can
+// The following classes are wrappers for memgraph::utils::MemoryResource, so that we can
 // use BENCHMARK_TEMPLATE
 
 class MonotonicBufferResource final {
-  utils::MonotonicBufferResource memory_{query::kExecutionMemoryBlockSize};
+  memgraph::utils::MonotonicBufferResource memory_{memgraph::query::kExecutionMemoryBlockSize};
```

Comment:
Since this is a cpp file is it possible to put all these under `memgraph` namespace, to avoid repetition of `memgraph::`? @jbajic if that's a hustle feel free to ignore it...